src/components/SlateEditor/plugins/file/SlateFileList.tsx

```diff
@@ -120,9 +119,6 @@ const SlateFileList = ({ element, editor, attributes, children }: Props) => {
   return (
     <FileListWrapper {...attributes} contentEditable={false}>
       <StyledHeaderWrapper>
-        <Heading fontWeight="bold" textStyle="heading.small" asChild consumeCss>
-          <h3>{t("files")}</h3>
-        </Heading>
         <HStack gap="3xsmall">
```

Comment:
Meningen å fjerne denne overskriften? isåfall burde vi vel fortsatt høyrestille knappene så blir det likt som andre steder!